kubernetes/schema_pod_spec.go

```diff
@@ -174,6 +174,43 @@ func podSpecFields(isUpdatable bool) map[string]*schema.Schema {
 			ValidateFunc: validateTerminationGracePeriodSeconds,
 			Description:  "Optional duration in seconds the pod needs to terminate gracefully. May be decreased in delete request. Value must be non-negative integer. The value zero indicates delete immediately. If this value is nil, the default grace period will be used instead. The grace period is the duration in seconds after the processes running in the pod are sent a termination signal and the time when the processes are forcibly halted with a kill signal. Set this value longer than the expected cleanup time for your process.",
 		},
+		"toleration": {
+			Type:        schema.TypeList,
+			Description: "Tolerations is an optional list of node tolerations controlling where pod can be placed",
+			Optional:    true,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"key": {
+						Type:        schema.TypeString,
+						Description: "Key for toleration effect",
+						Optional:    true,
+					},
+					"value": {
+						Type:        schema.TypeString,
+						Description: "Value for key",
+						Optional:    true,
+					},
+					"operator": {
```

Comment:
According to the docs the `operator` default to `Equal`. Should we be setting a default in the schema?
https://v1-8.docs.kubernetes.io/docs/api-reference/v1.8/#toleration-v1-core

Reply:
I think we should actually. I thought I had read it was not required, but looking back on it the sentence was about what happens on an empty key. Operator must be `Exists` in that case. I don't know how to do validation between keys, do you put that on the schema resource that contains both keys?

Reply:
Hmm, I'm not sure, but as far as I know it's not possible to setup Terraform config level validation between keys (except for the `ConflictsWith` attribute). Would have to be done in the resource Create or Update code.